tests/general/InvalidLinkBearTest.py

```diff
@@ -255,10 +257,26 @@ def response(status_code, *args, **kwargs):
             self.assertEqual([x.message
                               for x in list(uut.run('file', file_contents,
                                                     network_timeout=nt))], [])
+
+            with self.assertLogs(logging.getLogger()) as cm:
+                self.assertEqual([x.message
+                                  for x in list(uut.run('file', file_contents,
+                                                        timeout=20))], [])
+            self.assertEqual(cm.output,
```

Comment:
Also I'm not really a fan of the var name cm as it's not very descriptive. How about log?